nixos/modules/services/audio/spotifyd.nix

```diff
@@ -0,0 +1,46 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.spotifyd;
+  spotifydConf = pkgs.writeText "spotifyd.conf" cfg.config;
+in
+{
+  options = {
+    services.spotifyd = {
+      enable = mkEnableOption "spotifyd, a Spotify playing daemon";
+
+      dataDir = mkOption {
+        default = "spotifyd";
+        type = types.str;
+        description = "The directory where spotifyd stores its state, relative to /var/lib/.";
```

Comment:
do we need to make this configurable? Wonder about the usecases, given it's a unit with dynamic users and the "state" should be mostly caches, right?